source/setup-standard-clion-cmake.html.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Standard Configuration
+
+language_tabs:
+
+toc_footers:
+
+includes:
+
+search: true
+---
+
+# CLion CMake
+
+ 1. Once you've installed OpenSSL and cloned sleepy-discord into your project directory, open the project in CLion. 
+ 2. Click on Build > Build Project.
+ 3. Wait for the build process to finish.
+ 4. Find your CMake cache in sleepy-discord\cmake-build-debug\CMakeCache.txt.
```

Comment:
not sure why you need to find your CMake cache. after the build is finished the guide should end unless you are telling users how to run the program.